src/common/settings.js

```diff
@@ -459,9 +459,9 @@ const character_settings = {
         "type": "bool",
         "default": false
     },
-    "fighter-giant-might": {
-        "title": "Fighter: Giant Might",
-        "description": "Activate Giant Might to get advantage on Strength checks and saving throws and deal 1d6 extra damage",
+    "fighter-giants-might": {
```

Comment:
no reason to change the name of the option, that would invalidate whatever setting the user had before

Reply:
Good point, fixed